cmd/node/main.go

```diff
@@ -95,12 +95,6 @@ VERSION:
 		Usage: "The node will extract initial nodes info from the nodesSetup.json",
 		Value: "./config/nodesSetup.json",
 	}
-	// txSignSk defines a flag for the path of the single sign private key used when starting the node
```

Comment:
Since we removed these flags, should also update the scripts from /scripts subpackage

Reply:
done.